src/Microsoft.AspNetCore.SignalR.Core/IHubCallerClients`T.cs

```diff
@@ -3,12 +3,26 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A clients caller abstraction for a hub.
```

Comment:
"An abstraction that provides access to client connections, including the one that sent the current invocation" ?